README.md

```diff
@@ -332,6 +332,16 @@ The Platforms supported by the Node.js project are generally divided into four d
 
 *Deprecated* Platforms consist of prior *Primary*, *Secondary* or *Experimental* Platforms that are no longer under active development and are no longer supported by active Collaborators. Only the TSC can decide to move a *Primary* platform to *Deprecated* status. Support for *Deprecated* Platforms may be removed entirely after the next *semver-major* version increase.
 
+### Build Stability
+
+The stability of the Node.js Build process and infrastructure is critical for those who rely on the ability to build Node.js from source as part of their deployment workflow. The ability to support Node.js on multiple platforms requires that Collaborators avoid making breaking changes that require new or updated compiler versions, new C/C++ language level features, etc.
```

Comment:
Ok, so the question is: what do such changes mean to our platform and build stability? For instance, suppose we include linux/ppc in an LTS release, then v8 updates to a new compiler not currently supported by linux/ppc? We end up breaking on a supported platform. For `master` and normal release branches that ought to be fine, but we shouldn't allow an *LTS* release to happen until it's fixed.